PRINCIPLES.md

```diff
@@ -0,0 +1,34 @@
+# Principles for Building Ethical Conversational Assistants
```

Comment:
We don't need the `h1` tag in the document, because it is specified on the website side

Reply:
So I delete that line?

Reply:
Yes. I've added that title to the configuration in the website that will pull in the file